pkgs/by-name/fr/frr/package.nix

```diff
@@ -44,6 +47,7 @@
 , irdpSupport ? true
 , routeReplacementSupport ? true
 , mgmtdSupport ? true
+, grpcSupport ? true
```

Comment:
I'd recommend setting default to `false`, following https://docs.frrouting.org/en/latest/grpc.html saying
> It is currently disabled by default due its experimental stage

Reply:
Changed in e2d1572c6a70f7a841b8a069f04941182b014b49